.gitignore

```diff
@@ -1 +1 @@
-
+test.md
```

Comment:
- [ ] It is a requirement that the .gitignore file lists test.md file and node_modules directory (as it can be added by accident while setting up Linters). Kindly add the node_modules file to the .gitignore file as well

Reply:
Ok, great. Let's see if I got right now. Thanks!